### DISCUSSION TITLE
RPG Survival: Scavenge the Forsaken Farm, Craft and Survive Against the Moon Seekers!

### DISCUSSION THREAD

**Discussion by HomerN2763** [sunflower-land/sunflower-land#2975] [Ideas]
Remember the farmlands what the earliest players started their farming journey? Yeah, those before the Official 1.0 Release.

I was among the earliest players. I remembered displaying my allegiance to the Humans in the Humans vs. Goblins war.

It was the perfect opportunity for the Moon Seekers to invade the farmlands. Indeed, it was an effortless victory, since the strength of both the Humans and Goblins were at their weakest when the invaders came in.

Our farmlands were then overrun by them, and we all had to evacuate to the Sunflower Isles.

Months go by, and it was increasingly clear that the Moon Seekers didn't invade our old farmlands just for farming. According to recent intel, it seemed like discovered lots of valuable minerals and ancient relics deep underground, some of which are suspected to be literally out of this world.

As of now, we're already planning to build an Outpost on an island close enough to the Forsaken Farms, in which you can safely travel to the area without any hassle. Aside from that, we also plan on setting up Shops where you can purchase Consumables and Weapons, as well as Workshops where you can craft them as well.

Note, however, that the Moon Seekers don't welcome you there. And if you ever die along the way, you'll be returning empty-handed.

**Comment by TheDcol** [comment]
Sounds great especially from the lore point of view!
